### ISSUE TITLE
bug: Ionic 4 reorder breaks on tap in IOS

### ISSUE THREAD

**Issue by dalobstah** [ionic-team/ionic-framework#18628]
# Bug Report

**Ionic version:**

[x] **4.5.0**

**Current behavior:**

When tapping a reorder item on IOS a rendering bug occurs hiding most of the items. This only occurs with items wrapped in a reorder.

**Expected behavior:**

It is expected to do nothing when tapped like on other platforms.

**Steps to reproduce:**

Create a new project, create a reorder-group and use an item wrapped in a reorder, and then build for IOS. Tap one of the reorder items and they will disappear.

**Related code:**

HTML:
```html
<ion-reorder-group (ionItemReorder)="doReorder($event)" disabled="false">
    <ion-reorder *ngFor="let item of items">
        <ion-item>
            <ion-label>
                Item {{ item }}
            </ion-label>
        </ion-item>
    </ion-reorder>
</ion-reorder-group>
```

Typescript:
```typescript
public items: number[] = [
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8
];

public doReorder(e): void {
    this.items = e.detail.complete(this.items);
}
```

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.1.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.5.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms : ios 4.5.5
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.1, (and 4 other plugins)

Utility:

   cordova-res : 0.3.0 (update available: 0.5.1)
   native-run  : 0.2.0 (update available: 0.2.7)

System:

   ios-deploy : 1.9.4
   NodeJS     : v10.15.0 (/usr/local/bin/node)
   npm        : 6.9.0
   OS         : macOS Catalina
   Xcode      : Xcode 10.2.1 Build version 10E1001
```

**Comment by Jiroscopes** [comment]
+1

**Comment by JamieThomas-WKU** [comment]
+1

**Comment by justinelijahtaylor** [comment]
any workaround?

**Comment by dalobstah** [comment]
The strange behavior went away when wrapping the reorder around the icon, rather than the entire item. However, to drag, the icon must be pressed rather than just anywhere on item. It's not ideal, but at least it worked.

```html
<ion-reorder-group (ionItemReorder)="reorder($event)" disabled="false">
    <div *ngFor="let item of items" class="some-css">
        ... title ...
        <ion-reorder slot="end">
            ... icon ...
        </ion-reorder>
    </div>
</ion-reorder-group>
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. The behavior you are experiencing is related to a bug in WebKit, the engine that powers Safari on iOS. This issue impacts iOS 12-13 but is resolved in iOS 13.6 and newer.

I am going to close this as this is not a bug in Ionic Framework. To resolve this issue, I recommend updating your iOS devices to the latest version of iOS.
